2021-04-29 18:57:19,014 INFO     ---------------------------------------------------------------------------------------------
2021-04-29 18:57:19,014 INFO     Geo: cqd
2021-04-29 18:57:19,014 INFO     Data Path: data/FB15k-237-betae
2021-04-29 18:57:19,014 INFO     #entity: 14505
2021-04-29 18:57:19,014 INFO     #relation: 474
2021-04-29 18:57:19,014 INFO     #max steps: 1000
2021-04-29 18:57:19,014 INFO     Evaluate unoins using: DNF
2021-04-29 18:57:19,014 INFO     loading data
2021-04-29 18:58:01,466 INFO     Training info:
2021-04-29 18:58:01,472 INFO     Validation info:
2021-04-29 18:58:01,473 INFO     inp: 5000
2021-04-29 18:58:01,475 INFO     Test info:
2021-04-29 18:58:01,475 INFO     inp: 5000
2021-04-29 18:58:01,828 INFO     Model Parameter Configuration:
2021-04-29 18:58:01,828 INFO     Parameter embeddings.0.weight: torch.Size([14505, 2000]), require_grad = True
2021-04-29 18:58:01,828 INFO     Parameter embeddings.1.weight: torch.Size([474, 2000]), require_grad = True
2021-04-29 18:58:01,828 INFO     Parameter Number: 29958000
2021-04-29 18:58:06,499 INFO     Loading checkpoint models/fb15k-237-betae...
2021-04-29 18:58:06,800 INFO     tasks = inp
2021-04-29 18:58:06,801 INFO     init_step = 99999
2021-04-29 18:58:06,801 INFO     batch_size = 1000
2021-04-29 18:58:06,801 INFO     hidden_dim = 1000
2021-04-29 18:58:06,801 INFO     gamma = 12.000000
2021-04-29 18:58:06,801 INFO     Evaluating on Valid Dataset...
  0%|          | 0/5000 [00:00<?, ?it/s]
main.py --do_valid --do_test --data_path data/FB15k-237-betae -n 1 -b 1000 -d 1000 -lr 0.1 --disable_warmup --max_steps 1000 --cpu_num 0 --geo cqd --valid_steps 20 --tasks inp --print_on_screen --test_batch_size 1 --optimizer Adagrad --reg_weight 0.05 --log_steps 5 --checkpoint_path models/fb15k-237-betae --cqd d2 --cqd-t-norm prod --cqd-sigmoid --cqd-k 8 --no-save --cuda
overwritting args.save_path
logging to models/fb15k-237-betae
99999
Traceback (most recent call last):
  File "main.py", line 508, in <module>
    main(parse_args())
  File "main.py", line 497, in main
    valid_all_metrics = evaluate(model, valid_easy_answers, valid_hard_answers, args, valid_dataloader, query_name_dict, 'Valid', step, writer)
  File "main.py", line 165, in evaluate
    metrics = KGReasoning.test_step(model, tp_answers, fn_answers, args, dataloader, query_name_dict)
  File "/home/pminervi/workspace/KGReasoning/models.py", line 657, in test_step
    argsort = torch.argsort(negative_logit, dim=1, descending=True)
TypeError: argsort() received an invalid combination of arguments - got (NoneType, descending=bool, dim=int), but expected one of:
 * (Tensor input, name dim, bool descending)
      didn't match because some of the arguments have invalid types: (!NoneType!, !dim=int!, descending=bool)
 * (Tensor input, int dim, bool descending)
      didn't match because some of the arguments have invalid types: (!NoneType!, dim=int, descending=bool)

